src/query/graphite/native/builtin_functions.go

```diff
@@ -996,7 +996,7 @@ func totalBySum(seriesList []*ts.Series, index int) float64 {
 }
 
 // asPercent calculates a percentage of the total of a wildcard series.
-func asPercent(ctx *common.Context, input singlePathSpec, total genericInterface) (ts.SeriesList, error) {
+func asPercent(ctx *common.Context, input singlePathSpec, total genericInterface, _ ...int) (ts.SeriesList, error) {
```

Comment:
Can you implement `nodes` arg here?

With only defining this param it will only make this call not return an error when executed with this arg, but will return completely unexpected results.